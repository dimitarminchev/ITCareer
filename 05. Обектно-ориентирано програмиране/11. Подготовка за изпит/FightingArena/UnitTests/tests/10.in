CreateHero:Priest:GodPriest
CreateHero:Warrior:GodWarrior
CreateWeapon:GodAssassin:Bow:AssassinBow:5:5:5
HeroInfo:GodAssassin
CreateWeapon:GodWarrior:Sword:WarriorSword:5:5:5
CreateHero:Assassin:GodAssassin
CreateWeapon:GodPriest:MagicWand:PriestMagicWand:5:5:5
HeroInfo:SomeName
CreateWeapon:GodAssassin:Bow:AssassinBow:5:5:5
Fight:GodAssassin:GotinAssassin
Fight:GodAssassin:GodWarrior
CreateHero:Priest:GodPriest
Fight:GodWarrior:GodPriest
Fight:GodAssassin:GodPriest
Fight:Simo:GodPriest
HeroInfo:GodWarrior
CreateHero:Warrior:Pesho
Fight:GodPriest:Pesho
Fight:GodPriest:Pesho
CloseArena