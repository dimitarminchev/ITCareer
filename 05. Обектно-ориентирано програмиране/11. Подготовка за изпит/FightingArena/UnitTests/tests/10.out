Priest: GodPriest joined the Arena!
Warrior: GodWarrior joined the Arena!
Hero with name: GodAssassin does not exist!
Hero with name: GodAssassin does not exist!
Successfully armed hero GodWarrior with weapon Sword!
Assassin: GodAssassin joined the Arena!
Successfully armed hero GodPriest with weapon MagicWand!
Hero with name: SomeName does not exist!
Successfully armed hero GodAssassin with weapon Bow!
Hero with name: GotinAssassin does not exist!
Winner in the battle between GodAssassin and GodWarrior is GodAssassin with 20.00.
Hero with name: GodPriest already exists!
Winner in the battle between GodWarrior and GodPriest is GodPriest with 53.33.
Winner in the battle between GodAssassin and GodPriest is GodPriest with 33.33.
Hero with name: Simo does not exist!
Warrior: GodWarrior - Level: 0
Experience: 0
Sword: WarriorSword
  *Strength: 20
  *Agility: 5
  *Intelligence: 5
Power: 40.00
Warrior: Pesho joined the Arena!
Winner in the battle between GodPriest and Pesho is GodPriest with 93.33.
Winner in the battle between GodPriest and Pesho is GodPriest with 93.33.
Heroes: 4
Priest: GodPriest - Level: 1
Experience: 20
MagicWand: PriestMagicWand
  *Strength: 5
  *Agility: 5
  *Intelligence: 25
Power: 93.33
Assassin: GodAssassin - Level: 0
Experience: 30
Bow: AssassinBow
  *Strength: 10
  *Agility: 15
  *Intelligence: 5
Power: 60.00
Warrior: GodWarrior - Level: 0
Experience: 0
Sword: WarriorSword
  *Strength: 20
  *Agility: 5
  *Intelligence: 5
Power: 40.00
Warrior: Pesho - Level: 0
Experience: 0
Power: 0.00